recipe/migrations/cuda_112_ppc64le_aarch64.yaml

```diff
@@ -13,7 +13,7 @@ __migrator:
   check_solvable: false
   primary_key: cuda_compiler_version
   wait_for_migrators:
-    - aarch64andppc64leaddition
+    - ArchRebuild
```

Comment:
Oh interesting so it is based on [the class name]( https://github.com/regro/cf-scripts/blob/5a3d10f46132c7968bbd49c07dc30a9ea1d4e93f/conda_forge_tick/auto_tick.py#L548 ) then? Was thinking we wanted [this name]( https://github.com/regro/cf-scripts/blob/5a3d10f46132c7968bbd49c07dc30a9ea1d4e93f/conda_forge_tick/auto_tick.py#L551 ), but don't know the migrator code that well